CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# v1.6.2:
+
+### Features
+
+- [Feature #694](https://github.com/google/osv-scanner/pull/694)
+  Add subcommands! OSV-Scanner now has subcommands! The base command has been moved to `scan` (currently the only commands is `scan`).
+  By default if you do not pass in a command, `scan` will be used, so CLI remains backwards compatible.
```

Comment:
Can we maybe allude to guided remediation here as the motivation for this? Can link to our issue as well.

Reply:
Done!